linux_os/guide/system/software/gnome/gnome_screen_locking/dconf_gnome_session_idle_user_locks/tests/correct_value.pass.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# packages = dconf,gdm
+
+. $SHARED/dconf_test_functions.sh
+
+clean_dconf_settings
+add_dconf_lock "org/gnome/desktop/session" "idle-delay" "{{{ dconf_gdm_dir }}}" "00-security-settings-lock"
```

Comment:
The CI fail for this scenario is a legit concern, because, the OVAL and remediations expect the idle-delay item in /etc/dconf/db/local.d/locks but on some systems the dconf_gdm_dir variable is not local.d but gdm.d or distro.d. On RHEL 9 it's distro.d, so the test scenario inserts the strings into a different file than the rule expects.